ext/standard/proc_open.c

```diff
@@ -1182,6 +1231,33 @@ PHP_FUNCTION(proc_open)
 	childHandle = pi.hProcess;
 	child       = pi.dwProcessId;
 	CloseHandle(pi.hThread);
+#elif defined(USE_POSIX_SPAWN)
+	posix_spawn_file_actions_t factions;
+	int r;
+	posix_spawn_file_actions_init(&factions);
+
+	 if (close_parentends_of_pipes(&factions, descriptors, ndesc) == FAILURE) {
+		 posix_spawn_file_actions_destroy(&factions);
+		 close_all_descriptors(descriptors, ndesc);
+		 goto exit_fail;
+	 }
+
+	if(cwd)
+		 php_ignore_value(posix_spawn_file_actions_addchdir_np(&factions, cwd));
```

Comment:
Always use brackets (even for single statements):
```suggestion
	if (cwd) {
		php_ignore_value(posix_spawn_file_actions_addchdir_np(&factions, cwd));
	}	
```